### PR TITLE
Enhance README with various minor improvements

### PR DESCRIPTION
**Pull request by justinmayer**

### CODE REVIEW

**Comment by justinmayer** [comment]
And now that I think about it... I think we can remove the `invoke setup` task from README > Demo. Since Poetry is installed in a previous step, and since Pre-commit isn't needed to get the demo project running, I think perhaps this step is a bit superfluous.

**Comment by justinmayer** [comment]
@Natim: Any comments on this PR before I merge it?

File: README.rst [hunk 1]
```diff
@@ -23,69 +23,67 @@ Kagi
 
 Kagi provides support for FIDO WebAuthn security keys and TOTP tokens in Django.
 
-Kagi is a working proof-of-concept and isn't yet production ready. There are
-many TODOs sprinkled around the code that should be fixed before relying on it.
+Kagi is a working proof-of-concept and isn’t yet production-ready. There may be
```
Comment (Natim) [inline README.rst]: I would remove this sentence altogether. Hence there are no todo left in the code.

Comment (justinmayer) [reply README.rst]: Since the last remaining TODO in the base template will be resolved soon, I will remove this line and then merge this PR. Thanks!

**Pull request merged**